docs/docs/index.md

```diff
@@ -0,0 +1,17 @@
+# Welcome to MkDocs
```

Comment:
1. this file should be README.md
2. This folder should the default docs file for the whole demo project, not just MkDocs